src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -149,16 +143,8 @@ public String packageName(TargetLanguage language) {
     return packageName().get(language);
   }
 
-  /** Returns the type of the package */
   @Nullable
-  public abstract PackageType packageType();
-
-  /** Returns the type of the dependency */
-  @Nullable
-  public abstract DependencyType dependencyType();
-
-  @Nullable
-  public abstract GenerationLayer generationLayer();
```

Comment:
This looks like a breaking change, meaning we need to bump up the config schema version. With artman GA, I think we promised that we would backfill user config files if we made such breaking changes.

Reply:
This does not affect artman config, only toolkit config. Neither toolkit nor its config format are GA, so we should be ok here. Also nobody is using toolkit config directly (it is supposed to be generated by artman dynamically on generation step).

Reply:
Alright - as discussed offline, it's a generated intermediary config between artman & toolkit that isn't checked in anywhere. The consequence of this change basically means that after the push of artman & toolkit, you can't mix an older version of one & a newer version of the other.